arcade/draw_commands.py

```diff
@@ -731,6 +775,30 @@ def draw_lrtb_rectangle_filled(left: float, right: float, top: float,
     draw_rectangle_filled(center_x, center_y, width, height, color)
 
 
+def draw_lrbt_rectangle_filled(left: float, right: float, bottom: float, top: float, color: Color):
+    """
+    Draw a rectangle by specifying left, right, bottom and top edges.
+
+    :param float left: The x coordinate of the left edge of the rectangle.
+    :param float right: The x coordinate of the right edge of the rectangle.
+    :param float bottom: The y coordinate of the rectangle bottom.
+    :param float top: The y coordinate of the top of the rectangle.
+    :param Color color: The color of the rectangle.
+    :Raises AttributeError: Raised if left > right or top < bottom.
+    """
+    if left > right:
+        raise AttributeError(f"Left coordinate {left} must be less than or equal to the right coordinate {right}")
```

Comment:
These should be `ValueError`s in the new functions since we're breaking compatibility anyway.

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1650/commits/0af3ba990f9eacd0a13e188aa300615db4226460